### ISSUE TITLE
Add Album "Random to Temporary Playlist" isn't functional

### ISSUE THREAD

**Issue by SePHTaN** [ampache/ampache#2868]
### Describe the bug

When in Artist view clicking on the button "Random to Temporary Playlist" doesn't add the album to the playlist.

### To reproduce

1. Go to 'Artistview'
2. Click on 'Random to Temporary Playlist' icon

### Expected behavior
Album should be added random to the playlist.

## Environment

* Ampache version: 4.4.0 & develop
* Web server + version: nginx 1.14-2
* Server operating system: Debian 10
* Client operating system: Win10

### Client type

* Web

**Comment by lachlan-00** [comment]
looks like `%5B1%5D` are screwing up the link
```
/server/ajax.server.php?action=basket&type=album_random&id%5B1%5D=73443
```

**Comment by lachlan-00** [comment]
definitely that. the links generated don't have an id
![image](https://user-images.githubusercontent.com/1305249/113665561-02037d80-96f1-11eb-901d-160d420fcce4.png)

**Comment by lachlan-00** [comment]
fixed in develop e2d14eb55 but will need to do it a bit differently for master.

**Comment by lachlan-00** [comment]
fixed in master with 3286f6e5a
